bokehjs/src/coffee/models/annotations/label_set.coffee

```diff
@@ -22,13 +22,25 @@ export class LabelSetView extends TextAnnotationView
       @connect(@model.change, () ->
         @set_data(@model.source)
         @render())
+      @connect(@model.source.streaming, () ->
+        @set_data(@model.source)
+        @render())
+      @connect(@model.source.patching, () ->
+        @set_data(@model.source)
+        @render())
       @connect(@model.source.change, () ->
         @set_data(@model.source)
         @render())
     else
       @connect(@model.change, () ->
         @set_data(@model.source)
         @plot_view.request_render())
+      @connect(@model.source.streaming, () ->
+        @set_data(@model.source)
+        @plot_view.request_render())
+      @connect(@model.source.patching, () ->
+        @set_data(@model.source)
+        @plot_view.request_render())
```

Comment:
Currently not, but you wouldn't depend on this anyway. There is an intrinsic order of running callbacks (slots), which is in order of connection, but it's preferable that if you have a chain of actions happening in order for a given signal, you should state this explicitly like in this code. There are no other guarantees, so there may be interruptions between slots or even batches of slots running in different stack frames.